modules/sw360/sw360-workflow/src/main/java/org/eclipse/sw360/antenna/sw360/SW360MetaDataUpdater.java

```diff
@@ -92,4 +95,22 @@ public boolean isUploadSources() {
     public SW360Release uploadAttachments(SW360Release sw360Release, Map<Path, SW360AttachmentType> attachments) {
         return releaseClientAdapter.uploadAttachments(sw360Release, attachments);
     }
+
+    /**
+     * Creates a new {@code SW360Project} entity and sets typical properties
+     * before it is created.
+     *
+     * @param projectName    the project name
+     * @param projectVersion the project version
+     * @return the new, initialized {@code SW360Project} entity
+     */
+    private static SW360Project prepareNewProject(String projectName, String projectVersion) {
```

Comment:
suggestion: put this in a utils class (there are ones for release, component and attachment in the workflow package, so might not hurt to have one for project) 
i accept if you think it's unnecessary though

Reply:
The existing utility classes mainly seem to deal with the conversion between Artifacts and SW360 model objects. For this more or less trivial functionality I would rather keep the logic in the updater.

Reply:
That's fine with me